Documentation/reference/f5-ltm-lb.md

```diff
@@ -0,0 +1,92 @@
+# Using F5 BIG-IP LTM with Tectonic
+
+Installing Tectonic on bare metal or VMWare requires a master DNS (which resolves to any master node in the cluster), and a worker DNS (which resolves to any worker nodes for Ingress). Evaluation clusters may be set up using a DNS entry that points to each node, but production clusters require the use of a load balancer.
```

Comment:
first sentence: eliminate parens and s/which/that/:
"Installing Tectonic on bare metal or VMWare requires a master DNS record that resolves to any master node(s) in the cluster, and a worker DNS record which resolves to any worker nodes for Ingress."